security/compliance_operator/compliance-operator-release-notes.adoc

```diff
@@ -17,6 +17,72 @@ To access the latest release, see xref:../../security/compliance_operator/co-man
 
 For more information on compliance support for all Red{nbsp}Hat products, see link:https://access.redhat.com/compliance[Product Compliance].
 
+
+[id="compliance-operator-release-notes-1-7-0_{context}"]
+== OpenShift Compliance Operator 1.7.0
+
+The following advisory is available for the OpenShift Compliance Operator 1.7.0:
+
+* link:https://access.redhat.com/errata/RHBA-2025:3728-01[RHBA-2025:3728-01 - OpenShift Compliance Operator 1.7.0 bug fix and enhancement update]
+
+[id="compliance-operator-1-7-0-new-features-and-enhancements_{context}"]
+=== New features and enhancements
+
+* A `must-gather` extension is now available for the Compliance Operator installed on `aarch64`, `x86`, `ppc64le`, and `s390x` architectures. The `must-gather` tool provides crucial configuration details to Red Hat Customer Support and engineering. For more information, see xref:../../security/compliance_operator/co-support.adoc#compliance-must-gather_co-support[Using the must-gather tool for the Compliance Operator].
+
+* CIS Benchmark Support has been added to Compliance Operator 1.7.0. The profile supported is CIS OpenShift Benchmark 1.6.0 This update deprecates CIS OpenShift Profiles 1.4.0 and 1.5.0. For more information, see (link:https://issues.redhat.com/browse/CMP-3081[*CMP-3081*])
```

Comment:
the CIS Benchmark version should be 1.7.0. And there should be a be a full stop at the end of this sentence.
`The profile supported is CIS OpenShift Benchmark 1.7.0. `

```
ocp4-cis                   2m19s   1.7.0
ocp4-cis-1-4               2m20s   1.4.0
ocp4-cis-1-5               2m20s   1.5.0
ocp4-cis-1-7               2m20s   1.7.0
ocp4-cis-node              2m19s   1.7.0
ocp4-cis-node-1-4          2m19s   1.4.0
ocp4-cis-node-1-5          2m19s   1.5.0
ocp4-cis-node-1-7          2m19s   1.7.0
```